content/docs/add-react-to-a-website.md

```diff
@@ -1,202 +1,202 @@
+### İpucu: Üretim İçin JavaScript'i küçültün {#tip-minify-javascript-for-production}
 
-Before deploying your website to production, be mindful that unminifed JavaScript can significantly slow down the page for your users.
+Web sitenizi yayına sokmadan önce, küçültülmemiş JavaScript'in sayfanızı kullanıcılarınız için önemli ölçüde yavaşlatabileceğine dikkat edin.
 
-If you already minify the application scripts, **your site will be production-ready** if you ensure that the deployed HTML loads the versions of React ending in `production.min.js`:
+Uygulama komut dosyalarını küçültürseniz ve yayınlamaya hazır hale getirilen HTML'in de React'ın `production.min.js` içinde biten sürümlerini yüklediğinden emin olursanız **siteniz üretime hazır** olur:
 
 ```js
 <script src="https://unpkg.com/react@16/umd/react.production.min.js" crossorigin></script>
 <script src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js" crossorigin></script>
 ```
 
-If you don't have a minification step for your scripts, [here's one way to set it up](https://gist.github.com/gaearon/42a2ffa41b8319948f9be4076286e1f3).
+Komut dosyalarınız için bir küçültme adımınız yoksa, [ayarlamanın bir yolu budur](https://gist.github.com/gaearon/42a2ffa41b8319948f9be4076286e1f3).
 
-## Optional: Try React with JSX {#optional-try-react-with-jsx}
+## İsteğe bağlı: React'ı JSX ile deneyin {#optional-try-react-with-jsx}
 
-In the examples above, we only relied on features that are natively supported by the browsers. This is why we used a JavaScript function call to tell React what to display:
+Yukarıdaki örneklerde, yalnızca tarayıcılar tarafından doğal olarak desteklenen özelliklere itibar ettik. Bu yüzden React’a ne göstereceğini söylemek için bir JavaScript fonksiyon çağrısı kullandık:
 
 ```js
 const e = React.createElement;
 
-// Display a "Like" <button>
+// Bir "Beğen" <button>'u göster
 return e(
   'button',
-  { onClick: () => this.setState({ liked: true }) },
-  'Like'
+  { onClick: () => this.setState({ beğenildi: true }) },
+  'Beğen'
 );
 ```
 
-However, React also offers an option to use [JSX](/docs/introducing-jsx.html) instead:
+Bununla birlikte, React ayrıca bunun yerine [JSX](/docs/introducing-jsx.html) kullanma seçeneği de sunar:
 
 ```js
-// Display a "Like" <button>
+// Bir "Beğen" <button>'u göster
 return (
-  <button onClick={() => this.setState({ liked: true })}>
-    Like
+  <button onClick={() => this.setState({ beğenildi: true })}>
+    Beğen
   </button>
 );
 ```
 
-These two code snippets are equivalent. While **JSX is [completely optional](/docs/react-without-jsx.html)**, many people find it helpful for writing UI code -- both with React and with other libraries.
+Bu iki kod parçacığı eşdeğerdir. **JSX is [tamamen isteğe bağlı](/docs/react-without-jsx.html)** olsa da, birçok kişi UI (kullanici arayüzü) kodunu yazmak için yararlı bulmaktadır - hem React hem de diğer kütüphanelerle.
```

Comment:
Dil dinamiğimiz İngilizceden farklı olduğu için - den sonraki kısmı cümle içerisine alabilirsin. Örneğin:
JSX tamamen isteğe bağlı olsa da, hem React'i hem de diğer kütüphaneleri kullanan birçok kişi, kullanıcı arayüzü kodu yazmak için JSX'i yararlı bulmaktadır.